src/ui/server/controllers/insurance/all-sections/index.ts

```diff
@@ -18,7 +18,11 @@ export const get = (req: Request, res: Response) => {
 
   const flatApplicationData = flattenApplicationData(application);
 
-  const taskListStructure = generateGroupsAndTasks(application.referenceNumber, application.policyAndExport.policyType);
+  const taskListStructure = generateGroupsAndTasks(
+    application.referenceNumber,
+    application.policyAndExport.policyType,
+    application.exporterBroker.isUsingBroker,
+  );
```

Comment:
Can we update the unit tests for this? Surprised if it's not failing with this change

Reply:
its because that field is optional so not failing